setup.py

```diff
@@ -15,7 +15,7 @@
     py_modules=['scrapy_mongodb'],
     platforms=['Any'],
     install_requires=[
-        'pymongo >= 2.4.1'
+        'pymongo >= 2.4.1', 'six'
```

Comment:
I think we should specify which version of `six` should be installed.

Also, please move `six` to it's own line for better readability.